whenipress.d.ts

```diff
@@ -0,0 +1,49 @@
+declare module 'whenipress' {
+    type Handler = (event?: Event) => void;
+
+    type WhenIPressPlugin = Partial<{
+      mounted: (globalInstance: PendingKeyboardEventManager) => void;
+      bindingRegistered: (binding: string[], globalInstance: PendingKeyboardEventManager) => void;
+      bindingStopped: (binding: string[], globalInstance: PendingKeyboardEventManager) => void;
+      allBindingsStopped: (globalInstance: PendingKeyboardEventManager) => void;
+      beforeBindingHandled: (binding: string[], globalInstance: PendingKeyboardEventManager) => boolean | void;
+      afterBindingHandled: (binding: string[], globalInstance: PendingKeyboardEventManager) => void;
+    }>
+
+    type WhenIPressPluginOptions = Partial<{
+      urlsToSkip: string[];
+      skipAllUrls: boolean;
+    }>
+    
+    class PendingKeyboardEvent {
+      constructor(manager: PendingKeyboardEventManager, ...keys: string[]);
+      then(handler: Handler): this;
+      do(handler: Handler): this;
+      run(handler: Handler): this;
+      whenReleased(handler: Handler): this;
+      once(): this;
+      twiceRapidly(timeout?: number): this;
+      stop(): void;
+      createKeyDownListener(handler: Handler): void;
+      removeKeyDownListener(): void;
+      createKeyUpListener(handler: Handler): void;
```

Comment:
This has been replaced with `createKeyUpHandler`

Reply:
Noted